core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.component.ts

```diff
@@ -311,12 +311,8 @@ export class TopicsAndSkillsDashboardPageComponent {
           this.itemsPerPage), this.currentCount));
   }
 
-  getTotalCountValueForSkills(): number | string {
-    if (this.skillSummaries.length > this.itemsPerPage) {
-      return 'many';
-    }
-
-    return this.skillSummaries.length;
+  getTotalCountValueForSkills(): number {
+    return this.totalSkillCount;
```

Comment:
Hey @arjunrkaushik , the changes that you have made only show the number of skills instead of displaying "many" but I don't think this is what the issue is about. The issue is that the number of pages are not displayed in skill dashboard.

Reply:
Would you please confirm if this is the case?

Reply:
Hi @SanjaySajuJacob 
The issue was about "Skills" page showing the total number of skills as "many", which needed correction. It was not about the number of pages.
Lets assume the total number of skills as 120 and number of items per page as 20. The pagination displayed it as "1-20 of many", whereas the desired functionality was to display it as "1-20 of 120".
@seanlip Is that right?

Reply:
@arjunrkaushik It's nice to show the total number of skills, but on reading the original issue I agree with @SanjaySajuJacob. You're still welcome to include the fix for the former but that alone doesn't address the problem brought up in the issue.

Reply:
@seanlip Alright, will update the PR!

Reply:
@seanlip @SanjaySajuJacob Which one would you like me to keep?

1.
 ![image](https://user-images.githubusercontent.com/62321491/219768081-506f0a07-f921-46a4-8cde-0854093a6313.png)
2. 
![image](https://user-images.githubusercontent.com/62321491/219768184-947a03ac-baee-4e62-853c-f8f65dae2479.png)

Reply:
Personally, I think 1 looks neater than 2. @seanlip @kevintab95 , what would be your thoughts on this?

Also @arjunrkaushik, if you need a reviewer to respond, feel free to assign them to the PR, you can do this by leaving a comment in this format `{{Question/comment}} @{{reviewer_username}} PTAL` . Thanks!

Reply:
@arjunrkaushik Take a look at forum sites, google search, and others with lots of pages. You'll see that they use ellipses if the number of pages is too large, and show only a snippet around the current page (as well as the first and last page). I suggest you do that too.

Reply:
@seanlip @SanjaySajuJacob 
Have updated the code to make it look like this
![Screenshot 2023-02-18 180235](https://user-images.githubusercontent.com/62321491/219907131-2ed04940-70ec-44f0-91aa-9ab87d7eb4cf.png)
![Screenshot 2023-02-18 180301](https://user-images.githubusercontent.com/62321491/219907135-c5b70475-e55a-489e-9f1b-57dcd016225e.png)

Reply:
@arjunrkaushik It looks great to me, thanks!

Btw, if this isn't done for topics, could you do something similar for that too? That's optional but it might be worth fixing since it's similar. But overall, yup, this looks great!

Reply:
Alright will add it to topics as well!

Reply:
@seanlip Have added all changes as suggested!

Reply:
Hey @arjunrkaushik, the changes look good to me, would you please add the proof for the changes your have made in topic dashboard as well. Thanks!